### PR TITLE
home page update

### PR DESCRIPTION
**Pull request by maleck13**

### CODE REVIEW

File: src/index.njk [hunk 1]
```diff
@@ -30,7 +30,7 @@ tags: index-page # Note: This adds a custom class to layout.njk
...
+      <p>Leveraging the emerging ingress standard <a href="https://gateway-api.sigs.k8s.io/">Gateway API </a>, Kuadrant's features enable and empower cluster operators and application developers to work together to securely connect users with their services. Kuadrant empowers cluster administrators to deploy, manage, observe and protect gateways at scale giving them the confidence to allow application developers to self service knowing that endpoints exposed via these gateways are compliant, secure, visible and resilient.</p>
```
Comment (eguzki) [inline src/index.njk:33]: Kuadrant does not protect gateways, does it? At the data plane, rate limiting and authN/Z policies are meant to protect services, not gateways.

Comment (maleck13) [reply src/index.njk:33]: well I think that depends? If you have a rate limiting policy targeting a gateway that is setting a infra rate limit is that not protecting the gateway as a piece of that infrastructure

Comment (thomasmaas) [reply src/index.njk:33]: Maybe Eguzki’s point is that you always protect the thing 1 or more layers underneath of where you define the rate limit? As in, rate limit policy attached to a gateway protects the routes and the services. Rate limit policy attached to a route protects the service. So to protect a gateway you’d need a RateLimitPolicy at the layer above it?

Comment (maleck13) [reply src/index.njk:33]: ok new proposed wording https://github.com/Kuadrant/kuadrant.github.io/pull/28

Comment (eguzki) [reply src/index.njk:33]: > o to protect a gateway you’d need a RateLimitPolicy at the layer above it?

Yes, @thomasmaas !! Thanks for the explanation. I should have been more clear about that. 

PS: Routes are nothing else than configuration of the gateway. RLPs targeting a gateway can be seen as a **default** rate limiting policy for protecting all the services routed through the gateway. RLPs targeting a route (HTTPRoute) can be seen as the rate limiting rules for protecting only the services routed (can be multiple) from that specific route.

Comment (thomasmaas) [reply src/index.njk:33]: Agreed but still, 

- they are their own resource type
- the parent of a route doesn't have to be a gateway
- the parent of a route can be multiple gateways

**Pull request merged**